docs/cn/ufs/MapR-FS.md

```diff
@@ -71,16 +71,20 @@ ${ALLUXIO_HOME}/bin/alluxio fs mount /<path in Alluxio>/ maprfs:///<path in MapR
 
 完成所有相应配置后，你可以在本地启动Alluxio，观察一切是否正常运行：
 
-{% include Common-Commands/start-alluxio.md %}
+#{% include Common-Commands/start-alluxio.md %}
+{% include ./bin/alluxio format ./bin/alluxio-start.sh local %}
```

Comment:
I don't think this will work, please find other examples of people inlining includes, and follow that pattern